### PR TITLE
[docs] how to disable session replay

### PR DESCRIPTION
**Pull request by NSarfaty-Ddog**
### What does this PR do?
Indicates how to disable session replay

### Motivation
Multiple tickets where customer asked this question

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by MirandaKapin — CHANGES_REQUESTED** [review]
replaySampleRate to 0 
This removes the whole SKU, not just the feature. this should be clarified

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 1]
```diff
@@ -131,6 +131,11 @@ Expected Network bandwidth impact is less than 100Kb/min. Refined estimates will
 
 Session Replay follows the same 30 days retention policy as normal RUM sessions.
 
+### How to disable?
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/_index.md:134]:
```suggestion
### How do you disable Session Replay?
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 2]
```diff
@@ -131,6 +131,11 @@ Expected Network bandwidth impact is less than 100Kb/min. Refined estimates will
...
+
+- Remove startSessionReplayRecording()
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/_index.md:136]:
```suggestion
- Remove `startSessionReplayRecording()`.
```

File: content/en/real_user_monitoring/session_replay/_index.md [hunk 3]
```diff
@@ -131,6 +131,11 @@ Expected Network bandwidth impact is less than 100Kb/min. Refined estimates will
 
 Session Replay follows the same 30 days retention policy as normal RUM sessions.
 
+### How to disable?
+
+- Remove startSessionReplayRecording() 
+- Set replaySampleRate to 0 to stop collecting resources and long tasks that are added to the usage for this feature
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/_index.md:137]:
```suggestion
- Set `replaySampleRate` to `0` to stop collecting resources and long tasks, which are added to the usage for this feature.
```

**Review by maycmlee — CHANGES_REQUESTED** [review]

**Review by maycmlee — APPROVED** [review]

**Review by MirandaKapin — CHANGES_REQUESTED** [review]
- Remove `startSessionReplayRecording()` to stop session recordings
- Set `replaySampleRate` to `0` to stop collecting the RUM Session Replay plan which includes resources and long tasks.

**Pull request merged**